src/components/LoginForm.css

```
@@ -0,0 +1,91 @@
+  border-radius: var(--radius-sm, 8px);
+  padding: 2px 12px;
+  margin-bottom: 10px;
+  background: var(--surface, #ffffff);
+}
+
+.login-form__field ion-input {
+  --padding-start: 0;
+  --padding-end: 0;
+  --placeholder-color: var(--text-disabled, #b0b8c1);
+  --placeholder-opacity: 1;
+  font-size: 15px;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)